### ISSUE TITLE
Eclipse Plugin: Load SBuild JARs only once

### ISSUE THREAD

**Issue by lefou** [SBuild-org/sbuild#49]
Load the SBuild jars only once. Currently, it is newly loaded for each SBuild enabled project.

**Ticket imported form Redmine http://sbuild.tototec.de/sbuild**

**Original Redmine issue link:** [Redmine ID 49](http://sbuild.tototec.de/sbuild/issues/49)
**Reporter:** [Tobias Roeser](http://sbuild.tototec.de/sbuild/users/3) => @lefou
**Creating date:** 2012-10-23T00:46:26+02:00

**Assigned to:** [Tobias Roeser](http://sbuild.tototec.de/sbuild/users/3) => @lefou
**Start date:** 2012-10-20
**Due date:** 2013-04-25

 Another issue #45 relates this issue.
